app/views/partials/components-subnav.html

```diff
@@ -48,6 +48,11 @@
       link: '/components/header',
       selected: pageId == 'header'
     },
+    {
+      title: 'Branded Header',
```

Comment:
I'd suggest renaming it 'Header (Branded)' so that the subnav stays in alphabetical order, but keeps the two headers together.